tests/integration/test_structured_logging_json/configs/config_json.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0"?>
+<clickhouse>
+    <logger>
+        <!-- Structured log formatting:
+        You can specify log format(for now, JSON only). In that case, the console log will be printed
+        in specified format like JSON.
+        For example, as below:
+        {"date_time":"1650918987.180175","thread_name":"#1","thread_id":"254545","level":"Trace","query_id":"","logger_name":"BaseDaemon","message":"Received signal 2","source_file":"../base/daemon/BaseDaemon.cpp; virtual void SignalListener::run()","source_line":"192"}
+        To enable JSON logging support, just uncomment <formatting> tag below.
+        -->
+        <!-- <formatting>json</formatting> -->
```

Comment:
I suppose it should be uncommented to take effect
```suggestion
        <formatting>json</formatting>
```

Reply:
right, updated.